src/utils/IVeriCredential.ts

```diff
@@ -150,6 +150,7 @@ export interface DrivingPermit {
 	issueDate?: string;
 	fullAddress?: string;
 	issuingCountry?: string;
+	issueNumber?: string;
```

Comment:
is this needed?

Reply:
It was, but now it isn't so it's been removed